### DISCUSSION TITLE
Why is the latest version for Raspberry Pi so far behind the latest release?

### DISCUSSION THREAD

**Discussion by deandmx** [cockpit-project/cockpit#19808] [General]
Is there any reason for this. Is it possible to install the absolute latest version on the Pi?

**Comment by martinpitt** [comment]
You can install many operating systems on a RasPi, like e.g. [Debian](https://raspi.debian.net/tested-images/), [Ubuntu](https://ubuntu.com/tutorials/how-to-install-ubuntu-desktop-on-raspberry-pi-4) or [Fedora](https://docs.fedoraproject.org/en-US/quick-docs/raspberry-pi/). All of them have up to date cockpit packages.

I suppose you are running some older OS/Linux distro which just doesn't keep cockpit packages up to date. In some cases, like Raspian, you can just dist-upgrade to Debian, or install Debian packages.

**Reply by deandmx** [reply]
I'm running Debian GNU/Linux 11 (bullseye) though?

**Reply by martinpitt** [reply]
Right, so you have version 287 in backports. The current [Debian release](https://www.debian.org/releases/) is "12 bookworm" which has the most recent upstream cockpit.

**Reply by deandmx** [reply]
Plot twist! I also have a Pi 5 running bookworm and the latest version is still only 287.

**Reply by martinpitt** [reply]
bookworm-backports has version 308, see https://tracker.debian.org/pkg/cockpit
